pinot-core/src/main/java/org/apache/pinot/core/segment/creator/impl/SegmentDictionaryCreator.java

```diff
@@ -234,7 +259,7 @@ public int indexOfSV(Object value) {
     }
   }
 
-  public int[] indexOfMV(Object value) {
+  int[] indexOfMV(Object value) {
```

Comment:
why are we decreasing the scope

Reply:
I don't think the `public` scope is needed, hence removed but i'll revert that change.